test/test-files/appsync.ts

```diff
@@ -297,3 +297,65 @@ new AppsyncResolver(
     new aws_events.EventBus(stack, "");
   }
 );
+
+/**
+ * Unsupported - non-deterministic invocations
```

Comment:
Again: the word deterministic is wrong. This is the difference between static/variable length resolvers. Everything here is deterministic

Reply:
> In [mathematics](https://en.wikipedia.org/wiki/Mathematics), [computer science](https://en.wikipedia.org/wiki/Computer_science) and [physics](https://en.wikipedia.org/wiki/Physics), a deterministic system is a system in which no [randomness](https://en.wikipedia.org/wiki/Randomness) is involved in the development of future states of the system.

Well, not necessarily, I was considering non-deterministic to be a path with uses external inputs (or the result of invocations) where the logic of those invocations are unknown. (and could include "random" behavior).

I can use "unidirectional and defined statically" to match the AWS docs.